packages/@coorpacademy-components/src/template/back-office/dashboard-preview/index.js

```diff
@@ -56,16 +56,16 @@ const Dashboard = (props, context) => {
           className={style.dashboardNoSelection}
           style={{color: getOr('#00B0FF', 'common.primary', skin)}}
         >
-          {translate('Select a dashboard on the Sidebar')}
+          {translate('dashboard_select')}
```

Comment:
garde les clefs de trad en anglais. Ca fait office de fallback ..
`translate('Select a dashboard in the Sidebar')`

Reply:
:+1: